poleno/attachments/migrations/0003_content_type.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import magic
+
+from django.db import models, migrations
+
+def forward(apps, schema_editor):
+    Attachment = apps.get_model(u'attachments', u'Attachment')
+    mime = magic.Magic(mime=True)
+    for attachment in Attachment.objects.all():
+        content_type = mime.from_buffer(attachment.file.read())
```

Comment:
Nepouzivaj `attachment.file.read()` ale `attachment.content`. Pozri si implementaciu property `attachment.content`, ze preco treba pouzivat ju. Osetruje chybove stavy.

Reply:
@viliambalaz Ok, sorry. V migracii `attachment.content` nefunguje preto, ze v Djangu v migracii su dostupne ina samotne fieldy, nie vsak metody modelu. Tak teda v migracii citajme nadalej priamo z `attachment.file`

V `save` to nefunguje preto, ze `attachment.content` closne subor, do ktoreho vsak potom `super(Attachment, self).save(*args, **kwargs)` chce este zapisovat. Kaslime teda na to a pri savovani attachmentu tiez citajme data priamo z `attachment.file`.

Avsak kdekolvek inde v dalsom kode, kde budeme obsah attachmentu citat (mimo migracii), pouzivajme `attachment.content`